### PR TITLE
[PWGCF] Add weight output in flowGFWOmegaXi.cxx

### PR DESCRIPTION
**Pull request by fuchuncui**

### CODE REVIEW

File: PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx [hunk 1]
```diff
@@ -531,13 +575,17 @@ struct FlowGFWOmegaXi {
           registry.fill(HIST("InvMassK0s"), v0.pt(), v0.mK0Short(), v0.eta(), cent);
           registry.fill(HIST("hEtaPhiVtxzPOIK0s"), v0.phi(), v0.eta(), vtxz);
           fGFW->Fill(v0.eta(), fV0PtAxis->FindBin(v0.pt()) - 1 + ((fK0sMass->FindBin(v0.mK0Short()) - 1) * nV0PtBins), v0.phi(), wacc * weff, 8);
+          if (cfgOutputNUAWeights)
+            fWeightsK0s->Fill(v0.phi(), v0.eta(), vtxz, v0.pt(), cent, 0);
         }
       } else if (PDGCode == kLambda0) {
         if (TMath::Abs(v0.mLambda() - 1.115683) < cfgv0_mlambdawindow) {
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx:582]: Could you use the `MassLambda0` named constant  from `CommonConstants/PhysicsConstants.h` for the next iteration

**Review by victor-gonzalez — COMMENTED** [review]
Have a look at my comment

File: PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx (hunk 1 above)
Comment (fuchuncui) [reply PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx:582]: Thanks for your comment.

**Review by victor-gonzalez — APPROVED** [review]
Thanks!

**Pull request merged**